main/src/library/List.flix

```diff
@@ -1406,4 +1406,23 @@ namespace List {
             case x :: xs => loop(xs, ks -> k(consA(f(x), ks)))
         };
         loop(l, ks -> ks as & ef)
+
+    ///
+    /// Merges the two lists `l1` and `l2`. Assuming they are both sorted.
```

Comment:
Should it say something like, if the two elements compare `EqualTo`, then the elements of `l1` comes first? I'm not sure, if it matters to the user.

Reply:
It could say that.